### PR TITLE
allow beaver builder 2.6

### PR DESCRIPTION
**Pull request by vsolovei-smartling**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -18,39 +18,39 @@ pipeline {
...
             agent {
                 label 'master'
             }
 
             steps {
-                withCredentials([string(credentialsId: 'PROJECT_ID', variable: 'PROJECT_ID'), string(credentialsId: 'USER_IDENTIFIER', variable: 'USER_IDENTIFIER'), string(credentialsId: 'TOKEN_SECRET', variable: 'TOKEN_SECRET')]) {
-                    dir('.') {
-                        sh 'docker run --rm -w /plugin-dir -v $PWD:/plugin-dir -e MYSQL_HOST=localhost -e CRE_PROJECT_ID=$PROJECT_ID -e CRE_USER_IDENTIFIER=$USER_IDENTIFIER -e CRE_TOKEN_SECRET="$TOKEN_SECRET" wordpress-localization-plugin-php74:latest'
-                    }
+                dir('.') {
```
Comment (PavelLoparev) [inline Jenkinsfile:27]: Is it needed in this case? Seems like can be omitted since it's current directory

File: composer.json [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+  "repositories": [
+    {
+      "type": "github",
+      "url": "https://github.com/vsolovei-smartling/retry"
+    }
+  ],
   "require": {
     "galbar/jsonpath": "^2.0",
-    "jralph/retry": "^2.0",
+    "vsolovei-smartling/retry": "^2.0.1",
```
Comment (PavelLoparev) [inline composer.json:15]: I thought we already use your lib. i remember PR in the past

**Review by PavelLoparev — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (vsolovei-smartling) [reply Jenkinsfile]: Removed, thanks

File: composer.json (hunk 2 above)
Comment (vsolovei-smartling) [reply composer.json]: I'll revert the change for now, this actually won't work without #423

**Pull request merged**